src/ui/screens/home/ViewServers.qml

```diff
@@ -144,6 +153,25 @@ Item {
                             }
                         }
 
+        onSelectedSegmentChanged: {
+            root.telemetryScreenId = segmentedNav.selectedSegment.objectName === "tabSingleHop" ? "location_singlehop" : "location_multihop"
+
+            switch (segmentedNav.selectedSegment.objectName) {
+            case "tabSingleHop":
+                Glean.impression.locationSinglehopScreen.record({
+                    screen: root.telemetryScreenId,
+                });
+                break
+            case "tabMultiHop":
+                Glean.impression.locationMultihopScreen.record({
+                    screen: root.telemetryScreenId,
+                });
+                break
+            default:
+                console.log("Unrecognized selected segment")
```

Comment:
Consider recording an error value here instead of skipping telemetry. This pattern was adopted for other similar situations. See: https://mozilla-hub.atlassian.net/browse/VPN-5775. If you do it, also update the metric documentation on the YAML file.

Reply:
I see how this pattern is used for interaction metrics, which makes sense because the event is still happening, just from an unexpected `screen`. How would you go about doing something similar for impression metrics where the event doesn't even happen?

Reply:
Just do 
```
Glean.impression.locationMultihopScreen.record({
                    screen: "unexpected"
                });
```
The event is not supposed to happen, but the code path is there so theoretically it can happen. This is just a safe guard, I think it's the same as for interaction metrics at the end of the day.